docs: add caveats for commonjs modules transform (#2173)
babel/website: docs/plugin-transform-modules-amd.md

sidebar_label: transform-modules-amd
---

> This plugin transforms ECMAScript modules to [AMD](https://github.com/amdjs/amdjs-api/blob/master/AMD.md). The _syntax_ of Import/Export Statement (`import "./mod.js"`) and Import Expression (`import('./mod.js')`) are transformed to their AMD representation. Note that Babel is unaware of different resolution algorithms between implementations of ECMAScript modules and AMD.

Reviewer: Maybe:

```suggestion
This plugin transforms ECMAScript modules to [AMD](https://github.com/amdjs/amdjs-api/blob/master/AMD.md). Note that only the _syntax_ of import/export statements (`import "./mod.js"`) and import expressions (`import('./mod.js')`) is transformed, as Babel is unaware of the different resolution algorithms between implementations of ECMAScript modules and AMD.
```

?
